spk/bazarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = bazarr
+
+include ../../cross/bazarr/bazarr.mk
+
+SPK_VERS = $(BAZARR_VERSION)
+SPK_REV = 2
+
+SPK_ICON = src/bazarr.png
+
+REQUIRED_DSM = 6.1
+
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
+
+BETA = 1
+
+DEPENDS = cross/busybox
```

Comment:
There is no need to depend on busybox as python package already provides it

Reply:
Is it considered a good practice, though ? (Open question really)
Because, the day python removes busybox from its installation, this package would no longer be operational, right ?

Reply:
- [ ] Depend on busybox installation provided by python